src/Core/src/Hosting/FontCollectionExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.Maui.Hosting
+{
+	public static class FontCollectionExtensions
+	{
+		public static IFontCollection AddFont(this IFontCollection fontCollection, string filename, string? alias = null)
```

Comment:
Just one for now, but we could add support for adding embedded fonts and maybe some other things.